polars/polars-io/Cargo.toml

```diff
@@ -51,7 +54,7 @@ lexical-core = { version = "0.8", optional = true }
 memchr = "2.5"
 memmap = { package = "memmap2", version = "0.5.2", optional = true }
 num.workspace = true
-object_store = { version = "0.5.2", features = ["aws"], optional = true }
+object_store.workspace = true
```

Comment:
Is this still optional like this?

Reply:
I cannot figure the answer from the docs. I moved to explicitly defining the dependencies.